packages/grid/src/Grid.tsx

```diff
@@ -1907,6 +1907,12 @@ class Grid extends PureComponent<GridProps, GridState> {
       metrics.rowHeight
     );
 
+    // Check if at the top and attempting to scroll up
+    // Or at the bottom and attempting to scroll down
+    if ((top === 0 && deltaY < 0) || (top === lastTop && deltaY > 0)) {
```

Comment:
This seems to work, but I'm not the most familiar with how Grid was implemented. Would probably be good to check with @mofojed to make sure there aren't any edge cases we are missing here.

Reply:
ah, I think this can break horizontal scrolling when at top / bottom boundary since it's hard to scroll horizontally exactly without at least some deltaY